### PR TITLE
Updated the build scripts

### PR DESCRIPTION
**Pull request by julien51**
I am making the build scripts a bit simpler and removing all of the travis specific stuff so that it is easy to move to another provider

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: scripts/push-image.sh [hunk 1]
```diff
@@ -4,13 +4,10 @@
...
 IMAGE_NAME=$1
-ARGS=""
+IMAGE_TAG=$2
+
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+IMAGE_TO_PUSH="$DOCKER_REPOSITORY/$IMAGE_NAME:$IMAGE_TAG"
+docker tag $IMAGE_NAME $IMAGE_TO_PUSH
+docker push $IMAGE_TO_PUSH
 
-if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
```
Comment (julien51) [inline scripts/push-image.sh]: Moving this logic to travis.yaml for now.

File: .travis.yml [hunk 2]
```diff
@@ -35,10 +35,10 @@ script:
 after_success:
   - scripts/deploy.sh netlify
   - scripts/deploy-elasticbeanstalk.sh
-  - scripts/push-image.sh unlock
-  - scripts/push-image.sh unlock-integration
+  - test $TRAVIS_BRANCH = "master" && test $TRAVIS_PULL_REQUEST = "false" && scripts/push-image.sh unlock $TRAVIS_BRANCH
```
Comment (julien51) [inline .travis.yml]: Note that instead of pushing to `latest` we will be pushing to `master`.

Comment (akeem) [reply .travis.yml]: Are these setting the environment variables or checking for equality?

Comment (julien51) [reply .travis.yml]: Checking for equality ;)

**Comment by julien51** [comment]
I double checked the logs on travis and things are as expected: image was *not* pushed. Let's wait for a master build after this to ensure that the image has indeed been pushed.

**Pull request merged**